pkg/network/infraconfigurators/macvtap.go

```diff
@@ -73,9 +72,9 @@ func (b *MacvtapPodNetworkConfigurator) DiscoverPodNetworkInterface(podIfaceName
 	return nil
 }
 
-func (b *MacvtapPodNetworkConfigurator) podIfaceMAC() string {
-	if b.mac != nil {
-		return b.mac.String()
+func (b *MacvtapPodNetworkConfigurator) generateVmMac() string {
```

Comment:
I don't think we're generating anything - we're retrieving the mac either from the vmi spec or from the link.

Reply:
changed to `retrieveVmMac`.